UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/appsettings.json

```diff
@@ -40,6 +47,9 @@
     "ExchangeSetFileFolder": "V01X01",
     "EncRoot": "ENC_ROOT",
     "ReadMeFileName": "README.TXT",
+    "Info": "INFO",
+    "ProductFileName": "PRODUCTS.TXT",
+    "BaseCellExtension": ".000",
```

Comment:
Kindly add "BaseCellExtension": ".000" to appsettings.local.overrides.json to main consistency

Reply:
As per the Comment i have added "BaseCellExtension": ".000" to appsettings.local.overrides.json.